src/main/java/com/library/BookMapper.java

```diff
@@ -0,0 +1,50 @@
+package com.library;
+
+
+import com.library.model.Author;
+import com.library.model.Book;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+
+import java.util.List;
+import java.util.UUID;
+
+public class BookMapper {
+
+    private static final BookRepository repository = new BookRepository();
+    private static final String FIRST_NAME = "firstName";
+    private static final String LAST_NAME = "lastName";
+    private static final String TITLE = "title";
+    private static final String AUTHOR = "author";
+    private static final String ID = "id";
+
+    public static JSONArray mapBooks(List<Book> books) {
+        JSONArray jsonBooks = new JSONArray();
+        books.stream()
```

Comment:
public static JSONArray mapBooks(List<Book> books) {
        JSONArray jsonBooks = new JSONArray();
        books.stream()
            .map(BookMapper::mapBook)
            .forEach(jsonBooks::add);
        return jsonBooks;
/*        for (Book book: books) {
            JSONObject newBook = new JSONObject();
            newBook.put("title", book.getTitle());
            newBook.put("author", book.getAuthor());
            jsonBooks.add(newBook);
        }*/
    }

    private static JSONObject mapBook(Book book) {
        JSONObject newBook = new JSONObject();
        newBook.put(TITLE, book.getTitle());
        newBook.put(ID, book.getId().toString());
        JSONObject newAuthor = new JSONObject();
        newAuthor.put(FIRST_NAME, book.getAuthor().getFirstName());
        newAuthor.put(LAST_NAME, book.getAuthor().getLastName());
        newBook.put(AUTHOR, newAuthor);
        return newBook;
    }

Reply:
+